### PR TITLE
Mar 3468.3

### PR DESCRIPTION
**Pull request by Reshma-M-A**
Should now be able to see 

1. Team Email Address
2. Team Leads and Team Member's permission 
3. Team Briefs

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -67,6 +67,16 @@
...
               </form>
 
+              <form action="{{ url_for('.find_team_by_team_id') }}" method="get" class="question">
+                <label class="question-heading" for="team_id">Find Team by Team ID</label>
+                <p class='hint'>
+                  You can find this number at the end of the Teams’ URL.
+                </p>
+                <input type="text" name="team_id" id="team_id" class="text-box">
+                <input type="submit" value="Search" class="button-save">
+            </form>
```
Comment (malmas18) [inline app/templates/index.html]:
```suggestion
              </form>
```

File: app/templates/view_buyers.html [hunk 2]
```diff
@@ -39,6 +39,8 @@
               message = displayed_message,
               type = "destructive" if category == 'error' else "success"
           %}
+
+
```
Comment (malmas18) [inline app/templates/view_buyers.html]:
```suggestion
```

File: app/templates/view_buyers.html [hunk 3]
```diff
@@ -51,44 +53,67 @@
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
 
-{% if brief %}
+  {% if brief %}
+
```
Comment (malmas18) [inline app/templates/view_buyers.html]: need to fix the indentation

Comment (Reshma-M-A) [reply app/templates/view_buyers.html]: fixed

Comment (malmas18) [reply app/templates/view_buyers.html]: and everything else underneath it?

Comment (Reshma-M-A) [reply app/templates/view_buyers.html]: should be fixed now

**Pull request merged**